data/json/items/tool/workshop.json

```diff
@@ -1121,15 +1150,17 @@
     "description": "This is a small, sharp knife, designed for making precise cuts for textiles or crafts.  It could cause decent damage, but is difficult to hit things with it.  Its small, sharp blade allows for precision strikes in the hands of the skilled.  It is too small to butcher corpses with.",
     "weight": "23 g",
     "volume": "20 ml",
+    "longest_side": "10 cm",
     "price": 400,
     "price_postapoc": 100,
     "to_hit": -3,
     "cutting": 6,
     "material": [ "steel", "plastic" ],
     "symbol": ";",
     "color": "light_gray",
+    "techniques": [ "PRECISE" ],
```

Comment:
I suppose you didn't intend to make this change and it was a thing of and outdated master branch, since this was removed in #48352, merged 2 days ago

Reply:
You're totally right, that's the problem when you take more than 5 minutes to write and finish your PR 😋

Reply:
How do I approve your changes again?  Not familiar with GitHub.

Reply:
Oh, ummmm I actually can't remember hahaha, do you not see a "resolve conversation"?

Reply:
Resolve conversation I think just closes the conversation, I don't think it makes the commit.

Reply:
Ohhhh that's what you are searching, I didn't suggested a change in code directly, that's why you can't commit the changes I just made a suggestion in comment, you need to manually make the commit.

Reply:
Okay, there we go then.  So which two things need to be changed here specifically?

Reply:
I help you:
```suggestion
```

Reply:
Okay, I see what happened, for some reason it is listed as "add line" not red for "remove line" so I got confused.  Fixing the xacto knife!

Reply:
Yeah, that was my fault... Again, I suggested the changes in a comment (the green was for my comment in your addition), I should have just suggested the change properly.